ui/CMakeLists.txt

```diff
@@ -122,7 +123,7 @@ endif(HAVE_PRIVATE_QT_HEADERS)
 
 target_link_libraries(gammaray_ui
   LINK_PUBLIC ${QT_QTCORE_LIBRARIES} ${QT_QTGUI_LIBRARIES} gammaray_common
-  LINK_PRIVATE gammaray_common_internal gammaray_kitemmodels
+  LINK_PRIVATE gammaray_common_internal gammaray_kitemmodels gammaray_core
```

Comment:
We need objectidfilterproxymodel.h which is in core because... was trying to match objecttypefilterproxymodel.h.
Will move it to common then.